File: Project/frontend/fix_sidebar.js
const fs = require('fs');
const file = '/root/MedTrustX/Project/frontend/src/components/layout/Sidebar.tsx';
let content = fs.readFileSync(file, 'utf8');

// Replace occurrences where items list is missing ]}
content = content.replace(/},\n\];/g, '},\n  ]}\n];');

fs.writeFileSync(file, content);
console.log('Fixed missing braces in Sidebar.tsx');

File: Project/frontend/fix_sidebar.py
import re

file_path = '/root/MedTrustX/Project/frontend/src/components/layout/Sidebar.tsx'

with open(file_path, 'r') as f:
    content = f.read()

# Fix the missing brackets
content = re.sub(r'},\n\];', '},\n  ]}\n];', content)

with open(file_path, 'w') as f:
    f.write(content)

print('Fixed missing braces in Sidebar.tsx')
